tests/test_examples/test_gym_ex.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""The tests module contains the tests of the gym example."""
+
+import os
+import pytest
+import signal
+import subprocess
+import sys
+import time
+
+
+def test_gym_ex(pytestconfig):
+    """Run the gym ex sequence."""
+    if pytestconfig.getoption("ci"):
```

Comment:
when executing this test locally it fails .. due to an initialisation issue of the agent